deploy/test-environments/fleet_api/src/agents_enrolled.py

```diff
@@ -53,6 +67,40 @@ def verify_agents_enrolled() -> bool:
     return result
 
 
+def verify_agent_tags(agent, expected_agents) -> bool:
+    """
+    Verify that the agent has the expected tags
+    """
+    expected_tags = []
+    if agent.policy_id in expected_agents:
+        expected_tags = expected_agents[agent.policy_id].tags
+    for pattern in expected_tags:
+        pattern_exist = False
+        for tag in agent.tags:
+            if re.match(pattern, tag):
+                pattern_exist = True
+                break
+        if not pattern_exist:
+            logger.info(f"Agent {agent.id} does not have the expected tag {pattern}")
+            return False
```

Comment:
nit: Since this behavior is unexpected, it might be better to use `logger.warn()` or `logger.error()`